src/singleton.js

```diff
@@ -50,7 +51,8 @@ export default class Singleton extends Tracer {
     // Private and non-standard methods
     // ---------------------------------------------------------------------- //
 
-    /* For internal use only:
+    /**
```

Comment:
The comment intentionally did not use the `/**` syntax so that this function would not show up in the generated documentation (the `/**` style doc is what triggers to generate docs or not for the function)